impala/_rpc/thrift_util.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# Copyright (c) 2012 Cloudera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Thrift utility functions
+
+from thrift.transport.TSocket import TSocket
+from thrift.transport.TTransport import TBufferedTransport
```

Comment:
These classes actually need to be imported from the `_thrift_api` submodule.  Looking at this, it appears there's some unnecessary duplication there as well.  Perhaps this whole `thrift_util.py` file can be moved into `_thrift_api/__init__.py`?  And `TSocket` and `TBufferedTransport` can be moved into the common `__init__.py`?